src/libsyncengine/update_detection/update_detector/updatetree.cpp

```diff
@@ -91,7 +94,16 @@ std::shared_ptr<Node> UpdateTree::getNodeByPath(const SyncPath &path) {
     for (std::vector<SyncName>::reverse_iterator nameIt = names.rbegin(); nameIt != names.rend(); ++nameIt) {
         std::shared_ptr<Node> tmpChildNode = nullptr;
         for (const auto &childNode: tmpNode->children()) {
-            if (Utility::isEqualNormalized(*nameIt, childNode.second->name())) {
+            bool isEqual = false;
+            if (!Utility::checkEqualNormalized(*nameIt, childNode.second->name(), isEqual)) {
```

Comment:
We may consider a version of this function taking a logger as third argument to avoid code duplication.

Reply:
Not sure it's worth it for a 1 line and the function is called 2 times only.

Reply:
Will the function be called more often? 

There is the important point that refactoring the logging calls would cause the loss of the line information. There are certainly solutions for this. I just created a task to keep a record of this issue: https://infomaniak.atlassian.net/browse/KDESKTOP-1327

Let's postpone logging refactoring to the next sprint. Resolving.